watchman/utils.py

```diff
@@ -31,6 +34,13 @@ def import_string(dotted_path, error_prefix=''):
             return attr
 
 
-def get_checks(paths_to_checks):
-    for python_path in paths_to_checks:
+def get_checks(check_list=None, skip_list=None):
+    checks_to_run = frozenset(WATCHMAN_CHECKS)
+
+    if check_list is not None:
+        checks_to_run &= frozenset(check_list)
```

Comment:
`&=` and `-=` are pretty obscure Python operators, which most devs don't know. How about switching to `checks_to_run = checks_to_run.intersection(check_list)` and `checks_to_run = checks_to_run.difference(skip_list)` ?